omod/src/main/java/org/openmrs/module/fhirExtension/web/mapper/TaskMapper.java

```diff
@@ -121,8 +121,20 @@ private Concept getConceptForTaskType(String taskType) {
 			log.warn("Task type is not passed. Setting as null");
 			return null;
 		}
-		List<ConceptSearchResult> conceptsSearchResult = Context.getConceptService().getConcepts(taskType, getLocales(), false, null, null, null, null, null, 0, null);
-		List<Concept> conceptsByName = conceptsSearchResult.stream().map(ConceptSearchResult::getConcept).collect(Collectors.toList());
+		List<ConceptClass> parentConceptClasses = new ArrayList<ConceptClass>();
+		parentConceptClasses.add(Context.getConceptService().getConceptClassByName("ConvSet"));
+		List<ConceptSearchResult> conceptsSearchResult = Context.getConceptService().getConcepts("All Task Types", getLocales(), false, parentConceptClasses, null, null, null, null, 0, null);
```

Comment:
Can we extract this string `All Task Types` as a constant ? Also why do we need to pass Locales, we can keep it as `en` right becuase we have the name hardcoded in English locale

Reply:
updated as suggested